.cirrus.yml

```diff
@@ -333,9 +333,6 @@ static_alt_build_task:
     only_if: *not_docs
     depends_on:
         - build
-    # Community-maintained task, may fail on occasion.  If so, uncomment
-    # the next line and file an issue with details about the failure.
-    allow_failures: $CI == $CI
```

Comment:
Is this needed? The nix package is not something we committed to support officially.

Reply:
I reverted the change. The comment indicates that it was commented out, right?

Reply:
That line simply always evaluates "true" (it has to be done that funny way in cirrus).  It means we run the task, but basically ignore the results.  Commenting out that line (for example) allows a PR to be merged if (for some external reason) it's failing: We don't want that to block, e.g. a podman release.  So yes, it's fine to just comment out the `allow_failures` but please also leave the comment-there in case somebody needs it.